bundles/org.openhab.binding.cp750/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="cp750"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="connection">
+		<label>CP750 Binding Thing</label>
```

Comment:
Could you improve the label/ description for the Thing. Something like "CP750 Device"

Reply:
See new review pass, not yet resolved.

Reply:
Not yet resolved.